tests/ui-toml/array_size_threshold/array_size_threshold.stderr

```diff
@@ -1,29 +0,0 @@
-error: large array defined as const
-  --> $DIR/array_size_threshold.rs:4:1
-   |
-LL | const ABOVE: [u8; 11] = [0; 11];
-   | -----^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   | |
-   | help: make this a static item: `static`
-   |
-   = note: `-D clippy::large-const-arrays` implied by `-D warnings`
```

Comment:
Looks like the `ui-toml` tests aren't picking up the `clippy.toml`, they need to have `CLIPPY_CONF_DIR` (or `CARGO_MANIFEST_DIR` as in the current implementation) set to the directory they're in

Reply:
I cheated and made `clippy-driver` look in the directory of the file it's compiling first. I can look into using `CLIPPY_CONF_DIR`, but that would essentially restore the previous system of manually iterating over the folders and running the test suite once per file.